provisioning/vars/main.yml

```diff
@@ -5,4 +5,4 @@ _devtool_docroots:
   - "{{ php_xhprof_html_dir }}"
   - "{{ dashboard_install_dir|default('') }}"
 
-drupalvm_user: "{{ ansible_ssh_user | default(ansible_env.SUDO_USER, true) | default(ansible_env.USER, true) | default(ansible_user_id) }}"
+drupalvm_user: "{{ ansible_env.SUDO_USER | default(ansible_env.USER, true) | default(ansible_user_id, true) }}"
```

Comment:
Why does git delete that one space after `default(ansible_env.SUDO_USER, true)` when it doesn't need to? I almost thought it was some non-space space but hexdump seems correct.